dist/configs/config/map/default.cfg

```diff
@@ -5,9 +5,3 @@ bot names aliens "^9[bot]^1 Nolwen" "^9[bot]^1 Uriel" "^9[bot]^1 Morgan" "^9[bot
 
 // print a message on screen after 20 seconds
 delay 20000 cp "^2play fair, ^5play nice!"
-
-// fill server with 3 human bots and 3 alien ones,
```

Comment:
The `bot fill` commands should not be deleted without a trace since they serve as documentation that the command exists. How about leaving these here and making the `g_bot_minPlayers` a commented-out alternative?

Reply:
I would comment out this one instead. One of the idea is also that one may require less configuration files to get something working.